### ISSUE TITLE
Need mechanism to (or example of) setting PBC/box vectors

### ISSUE THREAD

**Issue by davidlmobley** [open-forcefield-group/smarty#196]
@jchodera - could you give a simple example of setting up periodic boundary conditions for OpenMM?

Currently, `ForceField.createSystem` uses whether or not box vectors are defined in the `Topology` to decide whether to apply periodic boundary conditions or not. However, there is no code here or in our documentation that sets box vectors, which means that any solvated systems we might set up with "default" behavior will end up without periodic boundary conditions. Someone would have to go in manually and add box vectors to the `Topology`, presumably.

The only way I've done this in OpenMM in the past is by using `tleap` to create an AMBER prmtop and crd for a solvated system and then using ParmEd to copy box vectors from that into an OpenMM system if needed. But here, we would want to avoid using `tleap`, so I'm not totally sure where else we would get these.

(I'm imagining, here, setting up a solvated system via `packmol`, such as via `SolvationToolkit`, then wanting to take the PDB file of the solvated system and apply a set of SMIRNOFF parameters and create a system for simulation with OpenMM).

**Comment by davidlmobley** [comment]
FTR, we're generating boxes using packmol and cubic boxes with dimensions running from 0 to (whatever) (see [openmoltools](https://github.com/choderalab/openmoltools/blob/master/openmoltools/packmol.py#L23-L28)) so presumably this results in rather simple box vectors which could be inserted into the `Topology`, I would imagine.

**Comment by jchodera** [comment]
I would append the `CRYST` header with the known periodic box vectors to the beginning of the PDB file output by packmol. See [here](https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/openmm/app/pdbfile.py#L283-L299) for format.

**Comment by davidlmobley** [comment]
Looks like it's already there: 
> CRYST1   31.892   31.892   31.892  90.00  90.00  90.00 P 1           1

Now I just need to figure out why it's not getting into my `Topology`.

**Comment by jchodera** [comment]
Here's the code that [reads the `CRYST1` header](https://github.com/pandegroup/openmm/blob/master/wrappers/python/simtk/openmm/app/internal/pdbstructure.py#L182-L189).

How are you loading your PDB file?

**Comment by davidlmobley** [comment]
Actually, it looks like I'm OK -- I had a system which was giving weird energies and I traced it to issues relating to PBC/cutoff, and thought it was PBC, but it seems that the PBC is actually set up properly (because `packmol` wrote the `CRYST1` header to the PDB and I was reading it using PDBFile). Instead, I just failed to notice that the default nonbonded scheme for `createSystem` here is `NoCutoff` so that's likely the issue. Marking this as closed. Thanks for the help.
